libparsec/crates/platform_device_loader/tests/recovery.rs

```diff
@@ -1,3 +1,43 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
-// TODO !
+#![allow(clippy::unwrap_used)]
+
+use libparsec_platform_device_loader::{export_recovery_device, import_recovery_device};
+use libparsec_tests_lite::parsec_test;
+use libparsec_types::HumanHandle;
+use libparsec_types::ParsecOrganizationAddr;
+use libparsec_types::UserProfile;
+use libparsec_types::{DeviceLabel, LocalDevice};
+
+#[parsec_test]
+async fn test_ok_recovery() {
+    let url = ParsecOrganizationAddr::from_any(
+        // cspell:disable-next-line
+        "parsec3://127.0.0.1:6770/Org?no_ssl=true&p=xCD7SjlysFv3d4mTkRu-ZddRjIZPGraSjUnoOHT9s8rmLA",
```

Comment:
Do we use the testbed here ? if thinking if the value of the URL is important and if it's value could change

Reply:
It only has to be a valid organization address